graphql/handler/transport/error.go

```diff
@@ -18,7 +18,10 @@ func SendError(w http.ResponseWriter, code int, errors ...*gqlerror.Error) {
 	if err != nil {
 		panic(err)
 	}
-	w.Write(b)
+	_, err = w.Write(b)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Hey, I am not sure about this change. If transient socket communications occur and this function panics, I'm not sure if that it will be recovered.

Reply:
Reverted.